src/components/stepper/StepperCircle.tsx

```diff
@@ -0,0 +1,28 @@
+import {MainContainer, Circle, Title, Step ,Img} from "./StepperCircle.styles"
+import CheckIcon from "../../assets/check.png"
+import Arrow from "../../assets/arrow.png"
+
+interface StepperCircleProps {
+    numberOfStep: number;
+    title: string;
+    currentStep: number;
+}
+
+function StepperCircle ({numberOfStep, title, currentStep}: StepperCircleProps){
+    
+    const isBackStep= numberOfStep< currentStep;
+    
+    const isCurrentStep=numberOfStep==currentStep
+    
+    return (
+        <MainContainer>
+            {isCurrentStep? 
+                <Img src={Arrow}></Img>:""}
```

Comment:
Esto podriamos tenerlo asi mejor


```suggestion
            { isCurrentStep && <Img src={Arrow} /> }
```

Estonces con esa "validacion" no te haria faltar devolver un `""`